src/extractor/restriction_parser.cpp

```diff
@@ -91,10 +91,20 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
         const std::string key(fi_begin->key());
         const std::string value(fi_begin->value());
 
+        // documented OSM restriction tags start either with only_* or no_*;
+        // check and return on these values, and ignore unrecognized values
         if (value.find("only_") == 0)
```

Comment:
Ah yes, see above. Yes we could check for all listed restrictions and check taginfo to make sure we're not missing some.